xcube-dev/xcube-viewer
# Change color and opacity of user places

We can now change color and opacity of user drawn points and shapes. Also, improved visual style of selected places in the map.

![image](https://github.com/user-attachments/assets/fa476fc3-a8c3-44a1-ad1a-b4585f3156ca)


Closes #216
Closes #97 

To verify:

- Draw several polygon places
- Create timeseries and statistics for them
- Next to the place selector find the new style editor
- Change color and opacity --> shape should change style accordingly
- See also the colors of timeseries and statistics changing
- Test for points too
